workflow-service/src/main/java/com/redhat/parodos/project/dto/ProjectUserRoleResponseDTO.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2022 Red Hat Developer
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.parodos.project.dto;
+
+import java.util.List;
+import java.util.UUID;
+
+import com.redhat.parodos.project.enums.Role;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * project user role response DTO
+ *
+ * @author Annel Ketcha (Github: anludke)
+ */
+
+@Builder
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class ProjectUserRoleResponseDTO {
+
+	private UUID id;
+
+	private String projectName;
+
+	private List<UserRoleResponseDTO> userResponseDTOList;
+
+	@Builder
+	@Data
+	@AllArgsConstructor
+	@NoArgsConstructor
+	public static class UserRoleResponseDTO {
```

Comment:
Can this class be defined in its own file?

Reply:
yes sure